nova_vm/src/engine/bytecode/executable.rs

```diff
@@ -461,6 +461,51 @@ fn is_chain_expression(expression: &ast::Expression) -> bool {
     }
 }
 
+trait ContainsExpression {
```

Comment:
**praise:** This is a nice impl to have <3

**suggestion:** This should maybe go in the syntax_directed_operations folder (or was it a module?), same place where `VarDeclaredNames` and friends live in.

Reply:
This was added in #401, not in the changes specific to this PR. I'll make a follow-up PR to move it.